src/js/options.js

```diff
@@ -186,6 +185,8 @@ var Header = {
 
 // --- Settings tab --- //
 
+var editModal;
+
```

Comment:
It would be better to put `editModal` inside `SettingsTab` (so for reference the modal you'll do `SettingsTab.editModal = something` instead of `editModal = something`.